front/app/components/admin/PostManager/components/LazyPostPreview/Idea/AdminIdeaEdit.tsx

```diff
@@ -100,6 +100,7 @@ interface State {
   processing: boolean;
```

Comment:
This AdminIdeaEdit component doesn't use JSON Forms or the json_forms_schema endpoint, which is strange. I think ideally we should try and replace this component with the IdeasEditPage so we only have two IdeaPage components to update when we're making changes. Having 3 is confusing as they have slightly different implementations.

Maybe something we can tackle during the upcoming idea form migration to the form builder :thinking:

Reply:
Indeed. The earlier we take out some of this old code, the faster we will be able to move. We can modify with the input work